common/features/config/networks/static/reducer.ts

```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       initial: 1
     }
   }
-};
+},
+
+  ETI: {
+    id: 'ETI',
+    name: 'Etherinc',
+    unit: 'ETI',
+    chainId: 101,
+    isCustom: false,
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'Etherinc Explorer',
+      origin: 'https://explorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
+      [SecureWalletName.SAFE_T]: ETI_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: ETI_DEFAULT,
```

Comment:
NO PULL REQUEST exists for Ledger.  Please remove LEDGER_NANO_S support from this PR until after you can demonstrate Ledger hardware wallet support.

Reply:
i am working on that

Reply:
i removed LEDGER_NANO_S for now